DigitalLearningSolutions.Web/Views/Supervisor/Index.cshtml

```diff
@@ -67,17 +67,28 @@
 <h2>Your to do list</h2>
 @if (Model.SupervisorDashboardToDoItems.Count() > 0)
 {
-  <ul class="nhsuk-contents-list__list">
-    @foreach (var toDoItem in Model.SupervisorDashboardToDoItems)
+<ul class="nhsuk-contents-list__list">
+  @foreach (var toDoItem in Model.SupervisorDashboardToDoItems)
+  {
+
+    if (toDoItem.SignOffRequest)
     {
       <li class="nhsuk-contents-list__item">
         <a class="nhsuk-contents-list__link nhsuk-link--no-visited-state" asp-action="ReviewDelegateSelfAssessment" asp-route-supervisorDelegateId="@toDoItem.SupervisorDelegateId" asp-route-candidateAssessmentId="@toDoItem.ID">
           Review and sign-off @toDoItem.DelegateName's @toDoItem.ProfileName
         </a>. Requested @toDoItem.Requested.ToShortDateString()
-
       </li>
     }
-  </ul>
+    else if(toDoItem.ResultsReviewRequest)
+    {
+      <li class="nhsuk-contents-list__item">
+        <a class="nhsuk-contents-list__link nhsuk-link--no-visited-state" asp-action="VerifyDelegateSelfAssessment" asp-route-supervisorDelegateId="@toDoItem.SupervisorDelegateId" asp-route-candidateAssessmentId="@toDoItem.ID">
```

Comment:
Could we use the same action and routing as the "Review and sign-off" link above, here. The link is broken for me with the current action and route data and should go to the same place.